2021/01/22 08:06:37 [crit] 1726#1726: *1 connect() to unix:/var/run/php/php7.0-fpm.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.0-fpm.sock:", host: "site"
2021/01/22 08:06:51 [crit] 1726#1726: *1 connect() to unix:/var/run/php/php7.0-fpm.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /admin HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.0-fpm.sock:", host: "site"
2021/01/22 08:29:42 [error] 2122#2122: *24 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/web/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 08:30:03 [error] 2122#2122: *24 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/web/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 08:30:14 [error] 2122#2122: *24 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/web/index.php HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 08:32:31 [error] 2122#2122: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/web/index.php HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 08:32:54 [error] 2122#2122: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 08:32:57 [error] 2122#2122: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 08:33:01 [error] 2122#2122: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/web/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 08:33:05 [error] 2122#2122: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 08:35:26 [error] 2122#2122: *38 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 08:35:29 [error] 2122#2122: *38 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/web/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 08:35:47 [error] 2122#2122: *38 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/web/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 08:37:40 [error] 2122#2122: *45 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/index.php HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 08:38:01 [error] 2122#2122: *45 open() "/app/backend/web/site/index" failed (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/site/index HTTP/1.1", host: "site"
2021/01/22 08:39:08 [error] 2122#2122: *51 open() "/app/backend/web/site/index" failed (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/site/index HTTP/1.1", host: "site"
2021/01/22 08:39:12 [error] 2122#2122: *51 open() "/app/backend/web/site/index" failed (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/site/index HTTP/1.1", host: "site"
2021/01/22 08:39:46 [error] 2122#2122: *51 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/web/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 08:44:43 [error] 2292#2292: *1 "/app/backend/web/backend/web/index.php" is not found (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/web/ HTTP/1.1", host: "site"
2021/01/22 08:44:47 [error] 2292#2292: *1 "/app/backend/web/backend/web/index.php" is not found (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/web/ HTTP/1.1", host: "site"
2021/01/22 08:44:51 [error] 2292#2292: *1 "/app/backend/web/backend/web/index.php" is not found (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/web/ HTTP/1.1", host: "site"
2021/01/22 08:47:07 [error] 2313#2313: *1 "/app/backend/web/backend/web/index.php" is not found (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/web/ HTTP/1.1", host: "site"
2021/01/22 08:47:10 [error] 2313#2313: *1 "/app/backend/web/backend/web/index.php" is not found (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/web/ HTTP/1.1", host: "site"
2021/01/22 08:54:57 [error] 2333#2333: *15 open() "/app/backend/web/site/login" failed (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/site/login HTTP/1.1", host: "site"
2021/01/22 08:55:23 [error] 2333#2333: *15 open() "/app/backend/web/site/login" failed (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/site/login HTTP/1.1", host: "site"
2021/01/22 08:55:26 [error] 2333#2333: *15 open() "/app/backend/web/site/login" failed (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/site/login HTTP/1.1", host: "site"
2021/01/22 08:55:45 [error] 2333#2333: *15 open() "/app/backend/web/index.php/site/login" failed (20: Not a directory), client: 192.168.83.1, server: site, request: "GET /backend/index.php/site/login HTTP/1.1", host: "site"
2021/01/22 08:55:49 [error] 2333#2333: *15 open() "/app/backend/web/index.php/site/login" failed (20: Not a directory), client: 192.168.83.1, server: site, request: "GET /backend/index.php/site/login HTTP/1.1", host: "site"
2021/01/22 08:55:59 [error] 2333#2333: *15 open() "/app/backend/web/index.php/site/login" failed (20: Not a directory), client: 192.168.83.1, server: site, request: "GET /backend/index.php/site/login HTTP/1.1", host: "site"
2021/01/22 08:56:08 [error] 2333#2333: *15 open() "/app/backend/web/index.php/site/login" failed (20: Not a directory), client: 192.168.83.1, server: site, request: "GET /backend/index.php/site/login HTTP/1.1", host: "site"
2021/01/22 08:56:16 [error] 2333#2333: *15 open() "/app/backend/web/index.php/debug/default/toolbar" failed (20: Not a directory), client: 192.168.83.1, server: site, request: "GET /backend/index.php/debug/default/toolbar?tag=600a933066c5c HTTP/1.1", host: "site", referrer: "http://site/backend/web/"
2021/01/22 09:03:29 [error] 2333#2333: *43 open() "/app/backend/web/index.php/debug/default/toolbar" failed (20: Not a directory), client: 192.168.83.1, server: site, request: "GET /backend/index.php/debug/default/toolbar?tag=600a94e1399d3 HTTP/1.1", host: "site", referrer: "http://site/backend/web/"
2021/01/22 09:04:02 [error] 2333#2333: *43 open() "/app/backend/web/debug/default/toolbar" failed (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/debug/default/toolbar?tag=600a9502b94c9 HTTP/1.1", host: "site", referrer: "http://site/backend/web/"
2021/01/22 09:08:17 [error] 2333#2333: *48 open() "/app/backend/web/debug/default/toolbar" failed (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/debug/default/toolbar?tag=600a96011194b HTTP/1.1", host: "site", referrer: "http://site/backend/web/"
2021/01/22 09:11:02 [error] 2333#2333: *51 open() "/app/backend/web/debug/default/toolbar" failed (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/debug/default/toolbar?tag=600a96a6ade2b HTTP/1.1", host: "site", referrer: "http://site/backend/web/"
2021/01/22 09:11:06 [error] 2333#2333: *51 open() "/app/backend/web/debug/default/toolbar" failed (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/debug/default/toolbar?tag=600a96aabedc7 HTTP/1.1", host: "site", referrer: "http://site/backend/web/"
2021/01/22 09:25:42 [error] 2502#2502: *1 "/app/backend/web/backend/index.php" is not found (2: No such file or directory), client: 192.168.83.1, server: site, request: "GET /backend/ HTTP/1.1", host: "site"
2021/01/22 11:58:13 [error] 2930#2930: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 11:58:46 [error] 2930#2930: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /baclend/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/22 11:58:53 [error] 2930#2930: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.83.1, server: site, request: "GET /backend/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "site"
2021/01/23 14:52:03 [error] 2972#2972: *1551 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/css/bootstrap.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcepti" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/css/bootstrap.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1552 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/css/site.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/css/site.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1556 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/31e71352/jquery.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Pag" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/31e71352/jquery.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1557 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page n" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1558 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.validation.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.validation.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1559 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/85e158a5/jquery.inputmask.bundle.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundH" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/85e158a5/jquery.inputmask.bundle.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1551 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.activeForm.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.activeForm.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1552 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/js/bootstrap.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/js/bootstrap.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1552 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/debug/default/toolbar". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page no" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/debug/default/toolbar?tag=600c381362371 HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1551 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/31e71352/jquery.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Pag" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/31e71352/jquery.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1551 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page n" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1551 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.validation.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.validation.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1551 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/85e158a5/jquery.inputmask.bundle.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundH" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/85e158a5/jquery.inputmask.bundle.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1551 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.activeForm.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.activeForm.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:03 [error] 2972#2972: *1551 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/js/bootstrap.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/js/bootstrap.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:52:29 [error] 2972#2972: *1552 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/css/site.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/css/site.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:29 [error] 2972#2972: *1551 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/css/bootstrap.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcepti" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/css/bootstrap.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:29 [error] 2972#2972: *1559 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/31e71352/jquery.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Pag" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/31e71352/jquery.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:29 [error] 2972#2972: *1558 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page n" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:29 [error] 2972#2972: *1557 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.validation.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.validation.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:29 [error] 2972#2972: *1556 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/85e158a5/jquery.inputmask.bundle.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundH" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/85e158a5/jquery.inputmask.bundle.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:29 [error] 2972#2972: *1552 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.activeForm.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.activeForm.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:29 [error] 2972#2972: *1551 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/js/bootstrap.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/js/bootstrap.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:29 [error] 2972#2972: *1556 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/debug/default/toolbar". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page no" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/debug/default/toolbar?tag=600c382d8fbe6 HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:29 [error] 2972#2972: *1557 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/31e71352/jquery.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Pag" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/31e71352/jquery.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:29 [error] 2972#2972: *1557 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page n" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:29 [error] 2972#2972: *1557 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.validation.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.validation.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:29 [error] 2972#2972: *1557 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/85e158a5/jquery.inputmask.bundle.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundH" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/85e158a5/jquery.inputmask.bundle.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:30 [error] 2972#2972: *1557 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.activeForm.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.activeForm.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:52:30 [error] 2972#2972: *1557 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/js/bootstrap.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/js/bootstrap.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:11 [error] 2972#2972: *1589 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/css/bootstrap.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcepti" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/css/bootstrap.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:11 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/31e71352/jquery.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Pag" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/31e71352/jquery.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:11 [error] 2972#2972: *1590 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/css/site.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/css/site.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:11 [error] 2972#2972: *1595 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page n" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:11 [error] 2972#2972: *1596 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.validation.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.validation.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:11 [error] 2972#2972: *1599 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/85e158a5/jquery.inputmask.bundle.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundH" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/85e158a5/jquery.inputmask.bundle.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:11 [error] 2972#2972: *1589 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.activeForm.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.activeForm.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:11 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/js/bootstrap.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/js/bootstrap.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:11 [error] 2972#2972: *1599 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/31e71352/jquery.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Pag" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/31e71352/jquery.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:12 [error] 2972#2972: *1589 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/debug/default/toolbar". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page no" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/debug/default/toolbar?tag=600c3893b692d HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:12 [error] 2972#2972: *1599 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page n" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:12 [error] 2972#2972: *1599 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.validation.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.validation.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:12 [error] 2972#2972: *1599 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/85e158a5/jquery.inputmask.bundle.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundH" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/85e158a5/jquery.inputmask.bundle.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:12 [error] 2972#2972: *1599 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.activeForm.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.activeForm.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:12 [error] 2972#2972: *1599 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/js/bootstrap.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/js/bootstrap.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1599 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/css/bootstrap.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcepti" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/css/bootstrap.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/31e71352/jquery.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Pag" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/31e71352/jquery.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1589 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/css/site.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/css/site.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1596 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page n" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1595 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.validation.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.validation.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1590 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/85e158a5/jquery.inputmask.bundle.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundH" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/85e158a5/jquery.inputmask.bundle.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/js/bootstrap.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/js/bootstrap.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1599 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.activeForm.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.activeForm.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1599 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/debug/default/toolbar". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page no" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/debug/default/toolbar?tag=600c38960befa HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/31e71352/jquery.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Pag" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/31e71352/jquery.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page n" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.validation.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.validation.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/85e158a5/jquery.inputmask.bundle.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundH" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/85e158a5/jquery.inputmask.bundle.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.activeForm.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.activeForm.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:14 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/js/bootstrap.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/js/bootstrap.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/site/index"
2021/01/23 14:54:33 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/css/bootstrap.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcepti" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/css/bootstrap.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1599 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/css/site.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/css/site.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1590 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/31e71352/jquery.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Pag" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/31e71352/jquery.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1595 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page n" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1596 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.validation.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.validation.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1589 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/85e158a5/jquery.inputmask.bundle.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundH" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/85e158a5/jquery.inputmask.bundle.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.activeForm.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.activeForm.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1599 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/js/bootstrap.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/js/bootstrap.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1599 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/debug/default/toolbar". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page no" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/debug/default/toolbar?tag=600c38a9241eb HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/31e71352/jquery.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Pag" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/31e71352/jquery.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page n" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.validation.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.validation.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/85e158a5/jquery.inputmask.bundle.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundH" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/85e158a5/jquery.inputmask.bundle.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/e2e59e6c/yii.activeForm.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpExcept" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/e2e59e6c/yii.activeForm.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 14:54:33 [error] 2972#2972: *1594 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "site/backend/assets/49f04cd/js/bootstrap.js". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/backend/assets/49f04cd/js/bootstrap.js HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/backend/"
2021/01/23 16:28:26 [error] 1405#1405: *19 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request: site/services in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#3 /app/frontend/web/index.php(17): yii\base\Applicati" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/services HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/site/index"
2021/01/23 16:28:35 [error] 1405#1405: *19 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request: site/about in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#3 /app/frontend/web/index.php(17): yii\base\Application-" while reading upstream, client: 192.168.83.1, server: site, request: "GET /site/about HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/site/index"
2021/01/23 17:04:15 [error] 1405#1405: *107 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "web/css/site.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found. in /app/v" while reading upstream, client: 192.168.83.1, server: site, request: "GET /web/css/site.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/news"
2021/01/23 17:05:32 [error] 1405#1405: *123 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "web/css/site.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found. in /app/v" while reading upstream, client: 192.168.83.1, server: site, request: "GET /web/css/site.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/news"
2021/01/23 17:06:53 [error] 1405#1405: *129 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "web/css/site.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found. in /app/v" while reading upstream, client: 192.168.83.1, server: site, request: "GET /web/css/site.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/news"
2021/01/23 17:06:58 [error] 1405#1405: *129 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "web/css/site.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found. in /app/v" while reading upstream, client: 192.168.83.1, server: site, request: "GET /web/css/site.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/news"
2021/01/23 17:07:21 [error] 1405#1405: *129 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "web/css/site.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found. in /app/v" while reading upstream, client: 192.168.83.1, server: site, request: "GET /web/css/site.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/news"
2021/01/23 17:07:30 [error] 1405#1405: *129 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "web/css/site.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found. in /app/v" while reading upstream, client: 192.168.83.1, server: site, request: "GET /web/css/site.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/news"
2021/01/23 17:15:34 [error] 1405#1405: *144 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "web/css/site.css". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found. in /app/v" while reading upstream, client: 192.168.83.1, server: site, request: "GET /web/css/site.css HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/news"
2021/01/24 07:06:27 [error] 1405#1405: *487 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\web\BadRequestHttpException: Unable to verify your data submission. in /app/vendor/yiisoft/yii2/web/Controller.php:209
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Controller.php(179): yii\web\Controller->beforeAction()
#1 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#2 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#3 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\" while reading upstream, client: 192.168.83.1, server: site, request: "POST / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/index"
2021/01/24 07:07:30 [error] 1405#1405: *499 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\web\BadRequestHttpException: Unable to verify your data submission. in /app/vendor/yiisoft/yii2/web/Controller.php:209
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Controller.php(179): yii\web\Controller->beforeAction()
#1 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#2 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#3 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\" while reading upstream, client: 192.168.83.1, server: site, request: "POST / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/index"
2021/01/24 07:09:21 [error] 1405#1405: *503 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\web\BadRequestHttpException: Unable to verify your data submission. in /app/vendor/yiisoft/yii2/web/Controller.php:209
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Controller.php(179): yii\web\Controller->beforeAction()
#1 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#2 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#3 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\" while reading upstream, client: 192.168.83.1, server: site, request: "POST / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/index"
2021/01/24 07:09:48 [error] 1405#1405: *509 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\web\BadRequestHttpException: Unable to verify your data submission. in /app/vendor/yiisoft/yii2/web/Controller.php:209
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Controller.php(179): yii\web\Controller->beforeAction()
#1 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#2 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#3 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\" while reading upstream, client: 192.168.83.1, server: site, request: "POST / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/index"
2021/01/24 07:10:37 [error] 1405#1405: *509 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\web\BadRequestHttpException: Unable to verify your data submission. in /app/vendor/yiisoft/yii2/web/Controller.php:209
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Controller.php(179): yii\web\Controller->beforeAction()
#1 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#2 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#3 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\" while reading upstream, client: 192.168.83.1, server: site, request: "POST / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/index"
2021/01/24 07:13:38 [error] 1405#1405: *518 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\web\BadRequestHttpException: Unable to verify your data submission. in /app/vendor/yiisoft/yii2/web/Controller.php:209
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Controller.php(179): yii\web\Controller->beforeAction()
#1 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#2 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#3 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\" while reading upstream, client: 192.168.83.1, server: site, request: "POST /index HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/index"
2021/01/24 11:13:06 [error] 1405#1405: *1001 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "web/img/feature-img1.png". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found. i" while reading upstream, client: 192.168.83.1, server: site, request: "GET /web/img/feature-img1.png HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/news"
2021/01/24 11:58:10 [error] 1405#1405: *1098 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "img/news-img1.png". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found. in /app/" while reading upstream, client: 192.168.83.1, server: site, request: "GET /img/news-img1.png HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/news"
2021/01/24 11:58:10 [error] 1405#1405: *1093 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "img/news-img2.png". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found. in /app/" while reading upstream, client: 192.168.83.1, server: site, request: "GET /img/news-img2.png HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/news"
2021/01/24 11:58:10 [error] 1405#1405: *1096 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "img/news-img3.png". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found. in /app/" while reading upstream, client: 192.168.83.1, server: site, request: "GET /img/news-img3.png HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site", referrer: "http://site/news"
2021/01/24 12:49:21 [error] 1405#1405: *1332 FastCGI sent in stderr: "PHP message: An Error occurred while handling another error:
yii\base\InvalidRouteException: Unable to resolve the request: site/error in /app/vendor/yiisoft/yii2/base/Controller.php:152
Stack trace:
#0 /app/vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction()
#1 /app/vendor/yiisoft/yii2/web/ErrorHandler.php(109): yii\base\Module->runAction()
#2 /app/vendor/yiisoft/yii2/base/ErrorHandler.php(135): yii\web\ErrorHandler->renderException()
#3 [internal function]: yii\base\ErrorHandler->handleException()
#4 {main}
Previous exception:
yii\base\InvalidRouteException: Unable to resolve the request "baclend/". in /app/vendor/yiisoft/yii2/base/Module.php:543
Stack trace:
#0 /app/vendor/yiisoft/yii2/web/Application.php(104): yii\base\Module->runAction()
#1 /app/vendor/yiisoft/yii2/base/Application.php(392): yii\web\Application->handleRequest()
#2 /app/frontend/web/index.php(17): yii\base\Application->run()
#3 {main}

Next yii\web\NotFoundHttpException: Page not found. in /app/vendor/yi" while reading upstream, client: 192.168.83.1, server: site, request: "GET /baclend/ HTTP/1.1", upstream: "fastcgi://unix:/var/run/php/php7.4-fpm.sock:", host: "site"
